hackathon_site/event/tests.py

```diff
@@ -96,3 +112,21 @@ def test_renders_when_logged_in(self):
         response = self.client.get(self.view)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertContains(response, "Dashboard")
+
+
+class LogOutViewTestCase(SetupUserMixin, TestCase):
+    """
+    Tests for the logout template. We simply redirect to the homepage.
+
+    This view uses django.contrib.auth.forms.AuthenticationForm, so no direct
```

Comment:
It doesn't use `AuthenticationForm`, or any forms. You can say it uses `django.contrib.auth.views.LogoutView`, so no view logic needs testing